src/calculateRentalCost.js

```diff
@@ -28,7 +28,26 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const price = 40;
+  const discontSmall = 20;
+  const discontBig = 50;
+  let finalPrice = 0;
```

Comment:
I mean that you can move the calculation to this variable and use it in your return statements, like `const calculation = something * something;`